### PR TITLE
Enable subscriptions schedule

### PR DESCRIPTION
**Pull request by imsickofmaps**

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@hodgestar Ready for review

File: subscription/tests.py [hunk 1]
```diff
@@ -32,3 +33,48 @@ def test_get_list_json(self):
...
+            "schedule": "/api/v1/periodic_task/10/", # Non existent task
+            "to_addr": "+271234",
+            "user_account": "80493284823"
+        }
+
+        response = self.api_client.post('/api/v1/subscription/', format='json',
+                                        authentication=self.get_credentials(),
+                                        data=data)
+        json_item = json.loads(response.content)
+        self.assertIn("error", json_item)
```
Comment (hodgestar) [inline subscription/tests.py]: Can we assert more about the error message than just that it exists and also check the http response code?

File: subscription/tests.py [hunk 2]
```diff
@@ -32,3 +33,48 @@ def test_get_list_json(self):
...
+
+    def test_post_subscription_good(self):
+        data = {
+            "contact_key": "82309423098",
+            "lang": "en",
+            "message_set": "/api/v1/message_set/3/",
+            "next_sequence_number": 1,
+            "resource_uri": "/api/v1/subscription/1/",
+            "schedule": "/api/v1/periodic_task/1/", # Non existent task
```
Comment (hodgestar) [inline subscription/tests.py]: I assume this task exists? :)

File: subscription/tests.py [hunk 3]
```diff
@@ -32,3 +33,48 @@ def test_get_list_json(self):
...
+        json_item = json.loads(response.content)
+        self.assertEqual("82309423098", json_item["contact_key"])
+        self.assertEqual(True, json_item["active"])
+        self.assertEqual(False, json_item["completed"])
+        self.assertEqual("en", json_item["lang"])
+        self.assertEqual("/api/v1/message_set/3/", json_item["message_set"])
+        self.assertEqual(1, json_item["next_sequence_number"])
+        self.assertEqual("/api/v1/periodic_task/1/", json_item["schedule"])
+        self.assertEqual("+271234", json_item["to_addr"])
+        self.assertEqual("80493284823", json_item["user_account"])
```
Comment (hodgestar) [inline subscription/tests.py:82]: Would it be possible to assert on the whole dict at once? So `.assertEqual(json_item, { ... }`?

File: subsend/tasks.py [hunk 4]
```diff
@@ -2,5 +2,5 @@
 
 
 @task()
-def add(x, y):
-    return x + y
+def process_message_queue():
+    return True
```
Comment (hodgestar) [inline subsend/tasks.py:6]: Is this a stub for later?

**Comment by hodgestar** [comment]
Otherwise looks good.

File: subscription/tests.py (hunk 3 above)
Comment (imsickofmaps) [inline subscription/tests.py:82]: No, due to timestamps etc.

File: subsend/tasks.py (hunk 4 above)
Comment (imsickofmaps) [inline subsend/tasks.py:6]: Yes

**Comment by imsickofmaps** [comment]
Other issues fixed - re-review ready

File: subscription/tests.py (hunk 3 above)
Comment (hodgestar) [inline subscription/tests.py:82]: Not for now, but one can just pop the timestamps and assert on the rest. It makes it much easier to see what's going on when a test fails if one asserts on the whole dict at once.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**